packages/action-group/README.md

```diff
@@ -349,3 +349,130 @@ The `justified` attribute will cause the `<sp-action-group>` element to fill the
     </sp-action-button>
 </sp-action-group>
 ```
+
+## Accessibility
```

Comment:
Nice! I'll pull this up locally later today t make sure it'll all copacetic, but this looks like an awesome expansion of the docs here.

If you're planning on making more contributions during the ongoing a11y review, we'd love to have you join the project so you can work from a branch (rather than a fork) which would give you full access to our CI system.